compiler-plugin-tests/src/test/resources/natural-programming-generic-negative/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+org = "np_test"
```

Comment:
We usually use `wso2` as the org name.

Reply:
I don't think we do (or should).

- https://github.com/ballerina-platform/module-ballerina-http/blob/master/ballerina-tests/http-interceptor-tests/Ballerina.toml
- https://github.com/ballerina-platform/module-ballerina-graphql/blob/master/ballerina-tests/graphql-interceptor-test-suite/Ballerina.toml